.github/workflows/extended_tests.yml

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2019, 2020 Alvar Penning
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: Run extended tests
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Send repository_dispatch event to dtn7-playground repo
+      run: |
+        curl -XPOST -u "${{ secrets.PAT_USERNAME }}:${{ secrets.PAT_TOKEN }}" -H "Accept: application/vnd.github.everest-preview+json"  -H "Content-Type: application/json" https://api.github.com/repos/dtn7/dtn7-playground/dispatches --data '{"event_type": "run"}'
```

Comment:
To get this working, I rebased the actions into the master branch within the dtn7-playground repository.
Afterwards I triggered it manually and the [action started](https://github.com/dtn7/dtn7-playground/actions/runs/1135011863).

Reply:
I guess we also could delete the core branch?

Reply:
Done.